doc/write_ex.md

```diff
@@ -0,0 +1,147 @@
+# How to write an exercise?
+
+Every input file begins with the required library calls, needed by the test
+function.
+
+```ocaml
+open Test_lib
+open Report
+```
+
+## Prelude and prepare
+
+Prelude and prepare functions are written almost as-if. You just write your
+function, with an added `%prelude`, or `%prepare` appended to the definition
```

Comment:
"with an added `%prelude" ... (to the end of the sence) -> "introduced by `let%prelude` or `let%prepare`."